src/log/middleware.rs

```diff
@@ -50,7 +50,8 @@ impl LogMiddleware {
         if status.is_server_error() {
             if let Some(error) = response.error() {
                 log::error!("Internal error --> Response sent", {
-                    message: format!("\"{}\"", error.to_string()),
+                    message: format!("{:?}", error),
```

Comment:
When using the `Debug` formatter on Nightly with `RUST_BACKTRACE` enabled, this will also print out with a backtrace.